lib/puppet/provider/cs_order/pcs.rb

```diff
@@ -19,37 +19,25 @@ def self.instances
     instances = []
 
     cmd = [ command(:pcs), 'cluster', 'cib' ]
-    raw, status = run_pcs_command(cmd)
+    raw, _ = run_pcs_command(cmd)
     doc = REXML::Document.new(raw)
 
     doc.root.elements['configuration'].elements['constraints'].each_element('rsc_order') do |e|
       items = e.attributes
 
-      if items['first-action']
-        first = "#{items['first']}:#{items['first-action']}"
-      else
-        first = items['first']
-      end
+      first = items['first']
+      second = items['then']
```

Comment:
Sounds good to me. Will we be able to retain support for the old syntax on the type level? That would be ideal, unless it becomes a huge code maintenance hassle.